domains/amaankazi.json

```diff
@@ -6,6 +6,8 @@
         "email": "amaankazi1793@gmail.com"
     },
     "record": {
-        "CNAME": "amaankazi.pages.dev"
+        "CNAME": "amaankazi.pages.dev",
```

Comment:
You cannot use CNAME with other records

Reply:
ok do i have to make a new pull request with another file now?